qm-plugins/qm-vip/class-qm-vip-output-html.php

```diff
@@ -26,8 +26,8 @@ public function output() {
 			<?php
 			if ( isset( $data['mu-plugins']['commit'] ) && isset( $data['mu-plugins']['date'] ) ) {
 				echo '<p><a href="https://github.com/automattic/vip-go-mu-plugins/commit/' . rawurlencode( $data['mu-plugins']['commit'] ) .
-				'" alt="GitHub URL of the commit that the stack was deployed from."><i><strong>Last modified: </strong>' . esc_html( $data['mu-plugins']['date'] ) .
-				'</i></a></p>';
+				'" aria-label="GitHub URL of the commit that the stack was deployed from."><i><strong><span class="screen-reader-text">Open in new tab </span>Last modified: </strong>' .
```

Comment:
Screen readers use `aria-label` to replace the whole of the anchor text - so screen reader users wouldn't hear the "Last modified: _date_" or the new "Open in a new tab" bit.
```suggestion
				'"><i><strong><span class="screen-reader-text">Open in new tab </span>Last modified: </strong>' .
```

Reply:
@GaryJones So if I'm understanding you correctly, remove `aria-label` then?

Reply:
Yes - it's rarely needed on links - the most common use for links is when it is more deeply describing one of a collection of links with repeated texr e.g. Read more, to make them sufficiently unique.

In this case, it was being used to describe where the link went, even though that's not something we would see for any other link. As it would have meant the loss of the Last modified: _date_ information to describe a GitHub commit URL, it is best to remove the attribute.